modules/gapi/include/opencv2/gapi/cpu/stereo.hpp

```diff
@@ -25,8 +25,8 @@ struct GAPI_EXPORTS StereoInitParam {
 
     int numDisparities = 0;
     int blockSize = 21;
-    double baseline = 70.;
-    double focus = 1000.;
+    double baseline = 63.5;
+    double focus = 3.6;
```

Comment:
I hope this change is consistent.

Reply:
Its a more relevant default value. @AsyaPronina asked for this changes